Core/LocalAdmin.cs

```diff
@@ -575,10 +567,21 @@ private void StartSession()
 
         RunScpsl();
     }
-
+    private static void ShowTitleScreen()
+    {
+        Console.ForegroundColor = ConsoleColor.Red;
+        Console.WriteLine(@"██╗░░░░░░█████╗░░█████╗░░█████╗░██╗░░░░░░█████╗░██████╗░███╗░░░███╗██╗███╗░░██╗
+██║░░░░░██╔══██╗██╔══██╗██╔══██╗██║░░░░░██╔══██╗██╔══██╗████╗░████║██║████╗░██║
+██║░░░░░██║░░██║██║░░╚═╝███████║██║░░░░░███████║██║░░██║██╔████╔██║██║██╔██╗██║
+██║░░░░░██║░░██║██║░░██╗██╔══██║██║░░░░░██╔══██║██║░░██║██║╚██╔╝██║██║██║╚████║
+███████╗╚█████╔╝╚█████╔╝██║░░██║███████╗██║░░██║██████╔╝██║░╚═╝░██║██║██║░╚███║
+╚══════╝░╚════╝░░╚════╝░╚═╝░░╚═╝╚══════╝╚═╝░░╚═╝╚═════╝░╚═╝░░░░░╚═╝╚═╝╚═╝░░╚══╝");
+        Console.ForegroundColor = ConsoleColor.Gray;
+    }
```

Comment:
Completely not needed. Can only potentially caused issues if executed on some specific operating systems/shells. Will only cause issues.